src/styles/App.css

```diff
@@ -240,3 +248,83 @@ form {
   width: calc(100% + 2rem);
   transform: translateX(-2rem);
 }
+
+.flyout {
+  position: fixed;
+  bottom: 1rem;
+  right: 1rem;
+  display: flex;
+  width: 10em;
+  border: 1px solid #bababa;
+  border-radius: 10px;
+  padding: 0 1em 1em;
+  overflow: hidden;
+  flex-direction: column;
+  gap: 1rem;
+  z-index: 1;
+  background-color: #fff;
+  box-shadow: inset 0 0 10px 0 #bababa30;
+  animation: flyout 0.5s ease-in alternate;
+}
+
+body.dark .flyout {
+  color: #f9f9f9;
```

Comment:
create css vars and for both themes and use it inside styles for specific elements
application is scalable and with your approach you have to create different styles for every element instead of using var